src/components/DpUploadFiles/index.ts

```diff
@@ -1,2 +1,2 @@
-export { default } from './DpUploadFiles.vue'
-export * from './DpUploadFiles.vue'
+export { default as DpUploadFiles } from './DpUploadFiles.vue'
+export { default as DpUpload } from './DpUpload.vue'
```

Comment:
do I get it right, and we can all refactor our exports to this to simplify things?

Reply:
If I understand correctly, we can remove this kind of export statement
`export * from './DpUploadFiles.vue'`
from our index files, since it exports the named exports from the component (which I believe we don't have)